pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -55,9 +57,8 @@ let
     postInstall = null;
```

Comment:
Part of the `postInstall` is needed again since the `OCCTWrapper.so` is required:
```nix
postInstall = ''
  mkdir -p "$out/lib"
  mv -v $out/bin/*.* $out/lib/
'';
```

Reply:
Thanks, I've integrated this, but apparently OCCTWrapper.so is not always present in $out/bin? I've changed the command so it doesn't fail the build, but ideally we should figure out which configurations require this step.

Reply:
I removed the postInstall step again - prePatch (inherited from prusa-slicer) already contains some logic that should handle this case. Can you check if it works with the current state of the PR?